package.json

```diff
@@ -69,6 +62,13 @@
     "@types/ember__test": "^4.0.1",
     "@types/ember__test-helpers": "^2.9.1",
     "@types/ember__utils": "^4.0.2",
+    "@types/ember-data": "^4.4.11",
```

Comment:
I don't understand why this moves around all the time 😄 seems like a mismatch between npm versions maybe?